ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -148,6 +150,13 @@ StatusDialog {
         }
     }
 
+    ModelEntry {
+        id: selectedSenderAccountEntry
+        key: "address"
+        sourceModel: popup.store.accounts
+        value: popup.store.walletSectionSendInst.selectedSenderAccountAddress
```

Comment:
The general idea for stores is to hide context properties and not expose them directly. So ideally we should avoid that in new code and hide objects injected from nim. This approach improves readability because once sth is exposed that way, there is no need to inspect nim code to discover e.g. what is exposed via `walletSectionSendInst`